components/filter-field/src/filter-field-tag/filter-field-tag.spec.ts

```diff
@@ -139,6 +152,156 @@ describe('DtFilterFieldTag', () => {
     editSubscription.unsubscribe();
     removeSubscription.unsubscribe();
   }));
+
+  describe('DtFilterTagOverlay', () => {
+    function setupComputedStyleMock(value: string | undefined): void {
```

Comment:
NIT: you can move this function to our testing library so that others can benefit from your implementation.